css/styles.css

```diff
@@ -126,10 +133,16 @@
 
 /* @group Column contents */
 
+.ab-column > ul {
+    padding: 0 20px 0 0;
+}
+.ab-column:last-of-type > ul {
+    border: 0;
+}
 .ab-column > ul > li {
     width: 330px;
     margin: 0 0 36px;
-    padding: 10px 0 10px 12px;
+    padding: 10px 0 12px 12px;
```

Comment:
This might appear to affect the alignment of checkboxes, while in fact it only adds a litte padding to the bottom of each option `li`. 
There is no reason to change this - so padding was restored from 2d193ccc36be60025bd5d8d82f6238b0203b5ffb

![feb -05-2017 22-32-50](https://cloud.githubusercontent.com/assets/1038789/22630082/214792c4-ebf3-11e6-80f4-d5fb4bd7522c.gif)